### PR TITLE
Mention multi-line strings in the syntax reference

### PR DESCRIPTION
**Pull request by notriddle**

### CODE REVIEW

File: lib/elixir/pages/Syntax Reference.md [hunk 1]
```diff
@@ -31,7 +31,26 @@ If the colon is followed by a double- or single-quote, the atom can be made of a
 
 ### Strings
 
-Strings in Elixir are written between double-quotes, such as `"foo"`. Any double-quote inside the string must be escaped with `\ `. Strings support Unicode characters and are stored in UTF-8 encoding.
+Single-line strings in Elixir are written between double-quotes, such as `"foo"`. Any double-quote inside the string must be escaped with `\ `. Strings support Unicode characters and are stored in UTF-8 encoding.
+
+Multi-line strings in Elixir are written with three double-quotes, and can have unescaped quotes within them. The resulting string will end with a newline. The indentation of the line with the initial `"""` on it is used to strip indentation from the inner string. For example:
```
Comment (josevalim) [inline lib/elixir/pages/Syntax Reference.md]: Indention is actually based on the last `"""`. :)

Comment (notriddle) [reply lib/elixir/pages/Syntax Reference.md]: Fixed

**Comment by notriddle** [comment]
Added multi-line charlists.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**